### PR TITLE
[DEV-1436] combined context

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: src/contexts/language.context.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  function changeLanguage(language: string) {
+    setLanguage(language);
+    i18n.changeLanguage(language);
+  }
+
+  const context = useMemo(
+    () => ({
+      language,
+      changeLanguage,
+      translate: (key: string, defaultValue: string) => t([key, defaultValue].join('.'), defaultValue),
```
Comment (davidleomay) [inline src/contexts/language.context.tsx:30]: Add option to add interpolation map?

File: src/components/navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+          <div
+            className={
+              isNavigationOpen ? 'absolute top-0 right-0 h-screen w-4/5 flex flex-col bg-dfxBlue-800' : 'hidden'
+            }
+          >
+            <div className="h-full w-full">
+              <div
+                className="w-full flex flex-row justify-end px-4 py-5 z-10"
+                onClick={() => setIsNavigationOpen(false)} // change isNavOpen state to false to close the menu
```
Comment (davidleomay) [inline src/components/navigation.tsx:29]: Comment is a bit useless 😆

File: src/components/navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                </li>
+                <li className="">
+                  <a href="/">Logout</a>
+                </li>
+                <li className="">
+                  <div onClick={() => changeLanguage('en')}>EN</div>
+                </li>
+                <li className="">
+                  <div onClick={() => changeLanguage('de')}>DE</div>
+                </li>
```
Comment (davidleomay) [inline src/components/navigation.tsx:71]: Use a language array somewhere

File: src/components/navigation.tsx (hunk 2 above)
Comment (Krysh90) [reply src/components/navigation.tsx:29]: that happens if a you copy something, from somewhere 😂

**Review by davidleomay — APPROVED** [review]

**Pull request merged**